.gitguardian.yaml

```diff
@@ -30,6 +30,8 @@ secret:
       name: Test secret mentioned in commit 6282ceb9035c1bfcceec5f6372f69063cdd588f7
     - match: e90eaa00d17c03309b660e1879c70b815509422c2d4cf0980b2e1221dffa0381
       name: Test secret mentioned in commit 6282ceb9035c1bfcceec5f6372f69063cdd588f7
+    - match: 6a1d11d5bbbb48f044bedfc2db5749e7fde3ad7e3f6a79601db034478a95f88a
+      name: Test secrets in snap_test_text_output.py
```

Comment:
nitpick: can you put this one with its friend from line 22, and use a similar name?